pkg/arvo/app/chat-view.hoon

```diff
@@ -122,10 +122,9 @@
     ++  truncate-envelopes
       |=  envelopes=(list envelope)
       ^-  (list envelope)
-      =/  length  (lent envelopes)
-      ?:  (lth length message-limit)
+      ?:  (lth (lent envelopes) message-limit)
         envelopes
-      (slag (sub length message-limit) envelopes)
+      (scag message-limit envelopes)
```

Comment:
You shouldn't need the length check here.  +scag will no-op if the list length is less than the supplied index.

Reply:
Cute, will make da change